CodeSniffer/DocGenerators/HTML.php

```diff
@@ -188,12 +188,13 @@ protected function printFooter()
     {
         // Turn off errors so we don't get timezone warnings if people
         // don't have their timezone set.
+        $errorLevel = error_reporting();
         error_reporting(0);
```

Comment:
Just do `$errorLevel = error_reporting(0);` instead of adding extra line because `error_reportring` function, if I'm not mistaken always returns previous error level.